src/pages/tezos/customizebot.js

```diff
@@ -363,6 +364,10 @@ const Customizer = () => {
 
   const [isUser] = useAtom(isUserAtom);
 
+  useEffect(() => {
```

Comment:
No need to specifically track a particular view. Amplitude does that by default.